aperturedb/cli/tokens.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+from typing import Annotated
+
+import typer
+
+from aperturedb.cli.console import console
+from aperturedb.Configuration import Configuration
+from aperturedb.CommonLibrary import create_connector
+from aperturedb.Connector import Connector
+
+app = typer.Typer()
+
+
+@app.command(help="List User Authentication Tokens")
+def list(user: Annotated[str, typer.Argument(help="The user the display tokens for")]):
+    conn = create_connector()
+    res, _ = conn.query([{"GetUserDetails": {"username": user}}])
+    utokens = res[0]['GetUserDetails']['tokens']
+    if len(utokens) == 0:
+        print(f"No Tokens for {user}")
+    else:
+        print(utokens)
+
+
+@app.command(help="Generate an Authentication token for a user")
+def generate(silent: Annotated[bool, typer.Argument(help="Only return token, don't print it.")] = False):
+    conn = create_connector()
+    token = generate_token(conn)
+    if not silent:
+        print(f"Created Access Token: {token}")
+    return token
+
+
+def generate_token(conn: Connector):
```

Comment:
I think a better place for these types of (reusable functions) would be in Utils.py . There might be more clients than just the cli bits.